woocommerce/woocommerce
# fixe-bug/Angola-currency

### All Submissions:

* [x ] Have you followed the [WooCommerce Contributing guideline](https://github.com/woocommerce/woocommerce/blob/master/.github/CONTRIBUTING.md)?
* [x ] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
* [ x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?


### Changes proposed in this Pull Request:
This PR proposes changing:

When the option angola is selected, the currency of the country is detected dynamically
Portugal was on the list of countries without the state field. Option removed
I came across this problem today when I installed woocommerce. When selecting the base of the store the app did not detect the currency of Angola. Before you could do it go through the huge list or the manual search.

Portugal was on the list of countries without the state field. Option removed.
Portugal was on the list of countries without the state field,  has been added states to portugal.

### How to test the changes in this Pull Request:

1. install/activate woocommerce
2. To choose Angola as Store Base
3.Then the current currency of Angola will automatically appear

### Other information:

* [ x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ x] Have you written new tests for your changes, as applicable?
* [ x] Have you successfully ran tests with your changes locally?


### Changelog entry

When the Angola option is selected, the country currency is detected dynamically
